### PR TITLE
Added Xray server service annotations

### PR DESCRIPTION
**Pull request by andreyshamray**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
Hi, guys! This PR will add a capability to add Xray server service annotations. In our case, it is needed to create tcp socket load balancer in AWS, because our ingress controller doesn't support that.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
Basically, it helps to resolve this issue: 
https://jfrog.com/knowledge-base/how-to-resolve-the-error-could-not-open-websocket-connection-websocket-could-not-find-connection-header-with-token-upgrade/

**Special notes for your reviewer**:
Will be good if you can merge it fast :) Thanks in advance!

### CODE REVIEW

**Comment by andreyshamray** [comment]
@eldada can you pls review this?
